### DISCUSSION TITLE
[Question]: Frigate+ free account and my submissions, and later the paid-for version

### DISCUSSION THREAD

**Discussion by ElectronicBattle** [blakeblackshear/frigate#12618] [Ask A Question] [answered]
### What is your question:

Hello All

I set up a free Frigate+ (F+) account a year or more ago and I got Frigate (F) working a couple of weeks ago.  When I first set up F+ I allocated nine cameras, of which four are now actively in use via F. I know I can't delete the unused cameras in the present incarnation of F+ but it is something that might be considered for the future. 

I am submitting images for analysis and so far I have done: 

![image](https://github.com/user-attachments/assets/0767e4fd-8f91-450a-bffd-74ce6cfeca40)

Some of the information for F+ says things like "you need to submit many hundreds of images" and elsewhere in the docs "more than 100 per camera" or similar wording.  Quite a lot, anyway.

(1) In my case, do I assume the number so far is regarded as being about 575/(4 actual cameras) = 140 images per camera or about 575/(9 listed cameras) = only 64?

(2) Is there a way for me/the maintainers/someone to see how my submissions have helped the overall modelling?  I hope my efforts are useful but I can't tell. I am being very careful to mark up  my submitted images so that everything in the shot is labelled (e.g. if there's a person stood next to a car but only the person has been identified, I'm tagging the car with a tight bounding box).I am submitting "this is a car" when it is correct, and "this is not  a cat" when it thinks it's a dog and I'm correcting the bounding box to "cat" etc.

(3) When I have submitted enough models to my free F+ account, what happens next? Do I get some form of notification or do I get a free model of (say) a dog but only based on my own cameras &  environments? If so, is it the case that the F+ paid-for model will be a more generic / more accurate/both "dog" based on everybody's F+ model inputs, numbering many thousand (rather than many hundred) ?

(4) Is my understanding correct here: https://docs.frigate.video/plus/ says that "person, face, car, license_plate, amazon, ups, fedex, package, dog, cat, deer" are the available models _at the moment_. The F+ "advantage" is that the F+ versions are trained on actual CCTV footage and therefore are higher accuracy than the base models from google which are in colour, or it's a studio shot, or it's ideal in some manner? 

I'll be joining F+ once I have got my somewhat unreliable "default" detection settings fine-tuned, which is what I am working on next.

Thanks

EB

**Comment by NickM-27** [comment, marked as answer]
1. The number is just a guideline for best initial results. Some users have requested a model after only a handful, there is no number that is guaranteed.
2. Not specifically, not every image that every user submits is included in the base model. 
3. There is no notification. The only way to get any model is to pay, that model starts with the base model (made up of images from the community) and then is fine tuned on your camera images. 
4. They are the available labels inside of the **model** at the moment. And yeah the advantage is it's purpose built and trained for this purpose. 

I think fine tuning your detection settings (assuming you mean object filters and masks) if you plan on using F+ is a wasted effort, the model has different scores and requires different values. You are better off running the model, using the example starting values in the plus docs then fine tuning from there.

**Reply by ElectronicBattle** [reply]
> 1. The number is just
> 2. Not specifically
> 3. There is no notification
> 4. They are the available 

OK, all understood.  I've joined! 

![image](https://github.com/user-attachments/assets/cf1242f9-4e68-4918-94da-1123d1c1b3ff)

> fine tuning your detection settings (assuming you mean object filters and masks)

Yes, I want to fine tune my motion detection (masks) at the moment. They  are OK but need some fiddling with; it's more the case that I can watch a fox walk right across the centre of the camera's field of view only six feet away and yet F only triggers about 50% of the time , whereas on another camera I can see a person 100 feet away in the far distance, occupying a tiny number of pixels and a very narrow FOV, and *I* can hardly make it out, but F reliably and accurately says "person" and it's correct.

**Another question** if I may: my main interest is cats and dogs and deer (existing labels and models) , and particularly foxes and badgers (not yet available).  F will reliably see a fox and say "dog" (most of the time) or "cat" the remainder of the time.  I have not been correcting it to say "this is not a dog/cat" (because I think that will have a detrimental effect on my (Frigate's) goal which is accurate species detection and differentiation).  Is my "don't correct" approach the best approach?

As an aside, when I created my subscription a few minutes ago I found a load of documentation pertaining to F+ which ordinarily has not been visible to me on the usual F+ landing page (see below). It is visible in the usual Frigate documentation structure of course. 

The F+ "need help" webpage button has never worked (Chromium) so I am assuming this is where the F+ docs link has been "hiding" from me. Might there need to be a specific link provided on the page, perhaps on the LHS above "settings" ? All of the other links on the page do  work.

![image](https://github.com/user-attachments/assets/fa0aa959-9bc1-4920-b7e9-a61766ea4690)
EB

**Reply by NickM-27** [reply]
> Another question if I may: my main interest is cats and dogs and deer (existing labels and models) , and particularly foxes and badgers (not yet available). F will reliably see a fox and say "dog" (most of the time) or "cat" the remainder of the time. I have not been correcting it to say "this is not a dog/cat" (because I think that will have a detrimental effect on my (Frigate's) goal which is accurate species detection and differentiation). Is my "don't correct" approach the best approach?

in general the recommendation is to avoid submitting those at all, but in any case once the new labels are supported you'll just want to change them to be the correct label

**Reply by ElectronicBattle** [reply]
> the recommendation is to avoid submitting those at all

Got it - I shall continue what I'm doing in that case. Thanks for the help once again.
